relecov_tools/read_lab_metadata.py

```
@@ -284,6 +284,17 @@ def adding_ontology_to_enum(self, m_data):
         """
         enum_dict = {}
         for prop, values in self.relecov_sch_json["properties"].items():
+            if not isinstance(values, dict):
```

This should raise an error, all properties in the schema should be dictionaries according to JsonSchema standards